project-base/app/config/cron.yaml

```diff
@@ -6,94 +6,78 @@ parameters:
 
 services:
     _defaults:
-        autowire: true
         autoconfigure: true
+        autowire: true
         public: false
 
-    # Services
+    # Data bridge import
 
-    Shopsys\FrameworkBundle\Model\Pricing\Vat\VatDeletionCronModule:
+    App\Component\DataBridge\DummyCronModule:
         tags:
-            - { name: shopsys.cron, hours: '*', minutes: '*', instanceName: service, readableName: 'Delete vats' }
+            - { name: shopsys.cron, hours: '*', minutes: '*/5', instanceName: dataBridgeImport, readableName: 'Dummy data bridge import' }
 
-    Shopsys\FrameworkBundle\Component\Cron\DeleteOldCronModuleRunsCronModule:
+    # Packetery
```

Comment:
in this file, I see comments indented by one line as well as not indented :thinking: 

```
    # Data bridge import

    App\Component\DataBridge\DummyCronModule:
```

vs

```
    # This module should run as last because it creates multiple kernels and fake requests.
    Shopsys\FrameworkBundle\Component\Error\ErrorPageCronModule:
```

I tried removing the empty lines below the comments and it does not report any issues so I would say it is not checked, right?

Reply:
the empty lines are somewhat weird, e.g. in `forms.yaml`, there are also some missing empty lines at the moment:

```

    App\Form\Admin\CustomerUserFormTypeExtension:
        tags:
            - { name: form.type_extension, extended_type: Shopsys\FrameworkBundle\Form\Admin\Customer\User\CustomerUserFormType }
    App\Form\Admin\Customer\BillingAddressFormTypeExtension:
        tags:
            - { name: form.type_extension, extended_type: Shopsys\FrameworkBundle\Form\Admin\Customer\BillingAddressFormType }

    App\Form\Admin\Customer\DeliveryAddressFormTypeExtension:
        tags:
            - { name: form.type_extension, extended_type: Shopsys\FrameworkBundle\Form\Admin\Customer\DeliveryAddressFormType }
```

Reply:
It was in previous version also, so I have manually updated it as we are not using standard for empty lines